services/item.go

```diff
@@ -9,7 +9,8 @@ func (file *File) writeItems(sheetName string, items []*models.Item) {
 }
 
 func (file *File) writeItem(sheetName string, item *models.Item) {
-	if item.Mode == "table" {
-		file.createTables(sheetName, item.StartingCellCoordinates, item.Tables)
+	file.createTables(sheetName, item.StartingCellCoordinates, item.Tables)
```

Comment:
What if there is no table ? The writer will crash without returning any errors or it will call this function for nothing.

We should check if the `item.Tables` exist and if it's not empty (use same check for `item.Graph`)